externs/olx.js

```diff
@@ -2302,6 +2302,21 @@ olx.interaction.DrawOptions.prototype.condition;
 
 
 /**
+ * @typedef {{features: ol.Collection.<ol.Feature>}}
+ * @api
+ */
+olx.interaction.TranslateOptions;
+
+
+/**
+ * The features the interaction works on.
+ * @type {ol.Collection.<ol.Feature>}
+ * @api
+ */
+olx.interaction.TranslateOptions.prototype.features;
```

Comment:
I think this should be optional. One may want to use a translate interaction that allows translating any feature on the map.